### PR TITLE
Added Bouncing Reflection Text Animation

### PR DESCRIPTION
**Pull request by Samar1110**
Issue : #9 

Added bouncing with Reflection Text Animation
@NiallEccles please review and merge my PR for Hactoberfest-2022

### CODE REVIEW

File: animations/bouncing-with-reflection.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  box-sizing: border-box;
+}
+
+body {
+  background-color:  #151719;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
```
Comment (NiallEccles) [inline animations/bouncing-with-reflection.css:14]: Can you merge these styles into your CSS class?

File: animations/bouncing-with-reflection.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  font-size: 60px;
+}
+.waviy span {
+  font-family: 'Alfa Slab One', cursive;
+  position: relative;
+  display: inline-block;
+  color: #fff;
+  text-transform: uppercase;
+  animation: waviy 1s infinite;
+  animation-delay: calc(.1s * var(--i));
```
Comment (NiallEccles) [inline animations/bouncing-with-reflection.css:27]: Not really sure if this variable is supposed to be a CSS variable or comes from SASS

File: animations/bouncing-with-reflection.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+.waviy {
+  position: relative;
+  -webkit-box-reflect: below -20px linear-gradient(transparent, rgba(0,0,0,.2));
+  font-size: 60px;
+}
+.waviy span {
```
Comment (NiallEccles) [inline animations/bouncing-with-reflection.css:20]: You should remove the span selector and just have your class name.

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Thanks for making this PR.

I've highlighted a few changes for you to make before I can merge!

File: animations/bouncing-with-reflection.css (hunk 1 above)
Comment (Samar1110) [reply animations/bouncing-with-reflection.css]: Merged the Styles

**Comment by NiallEccles** [comment]
Nice!

**Pull request merged**